cli/configssh.go

```diff
@@ -207,6 +218,10 @@ func (r *RootCmd) configSSH() *clibase.Cmd {
 			r.InitClient(client),
 		),
 		Handler: func(inv *clibase.Invocation) error {
+			if sshConfigOpts.waitEnum != "auto" && skipProxyCommand {
+				return xerrors.Errorf("cannot specify both --skip-proxy-command and --wait")
+			}
```

Comment:
Makes sense, but for anyone reading this code for the first time, maybe include a comment on this that says something like:
```golang
// The waitEnum is a flag applied to the ProxyCommand. If the user specifies skipProxyCommand
// then the waitEnum flag is not used.
```

Just took me a second to remember what SkipProxyCommand did.